lib/shared/models/selection-state.js

```diff
@@ -65,15 +65,19 @@ exports.toggleDrive = (driveDevice) => {
  * This is a temporary function during the transition to multi-writes,
  * remove this and its uses when multi-selection should become user-facing.
  *
- * @param {String} driveDevice - drive device identifier
+ * @param {String} device - drive device identifier
  *
  * @example
+ * console.log(selectionState.getSelectedDevices())
+ * > [ '/dev/disk1', '/dev/disk2', '/dev/disk3' ]
  * selectionState.deselectOtherDrives('/dev/disk2')
+ * console.log(selectionState.getSelectedDevices())
+ * > [ '/dev/disk2' ]
  */
-exports.deselectOtherDrives = (driveDevice) => {
-  if (exports.isSelectedDrive(driveDevice)) {
-    exports.deselectAllDrives()
-    exports.selectDrive(driveDevice)
+exports.deselectOtherDrives = (device) => {
```

Comment:
Umm, perhaps I'm missing something, but why does the `@example` for `deselectOtherDrives` show examples of `getSelectedDevices` ?

Reply:
It's to clarify what exactly it does. Do you think it's too verbose?

Reply:
Sorry, my fault. I was misreading the diff and hadn't spotted that you'd still got `deselectOtherDrives` in the middle of the example :laughing: 
![image](https://user-images.githubusercontent.com/476186/36600682-d6b346b4-18aa-11e8-9f73-43ca1fbb8f38.png)